$NetBSD: patch-aa,v 1.1 2001/07/01 18:36:04 dmcmahill Exp $

add a "Quit" button.  Change will be submitted to the author.

--- mpac.tcl.orig	Tue Jan 18 21:25:34 2000
+++ mpac.tcl	Tue May 15 16:14:05 2001
@@ -195,5 +195,6 @@
 button .two.calculate -text "Recalculate" -command CalcIt -padx 5
 button .two.reset -text "Reset" -command reset_values -padx 5 
-pack .two.calculate .two.reset	-side right
+button .two.quit -text "Quit" -command exit -padx 5 
+pack .two.calculate .two.reset .two.quit       -side right
 
 
